businessRules/ideas.js

```diff
@@ -1,59 +1,50 @@
-const { getAllIdeas, createIdea, updateIdea } = require('../models/ideas');
+const { getAllIdeas, createNewIdea, updateIdea } = require('../models/ideas');
```

Comment:
Is there a reason why `getAllIdeas` is not called simply `getIdeas`?

Reply:
There was, but it doesn't exist no more, so... Renamed. )